addons/policies/bootstrap_entity_policies.json

```diff
@@ -2257,7 +2260,8 @@
                 [
                     "admin",
                     "service-account-atlan-argo",
-                    "service-account-atlan-backend"
+                    "service-account-atlan-backend",
+                    "atlan-governance-workflows"
```

Comment:
No need of this, you can remove this change

Reply:
Alright, will remove in the revision.